proxy/src/main/java/com/velocitypowered/proxy/connection/util/ServerListPingHandler.java

```diff
@@ -48,7 +48,9 @@ private ServerPing constructLocalPing(ProtocolVersion version) {
         new ServerPing.Version(version.getProtocol(),
             "Velocity " + ProtocolVersion.SUPPORTED_VERSION_STRING),
         new ServerPing.Players(server.getPlayerCount(), configuration.getShowMaxPlayers(),
-            ImmutableList.of()),
+            server.getAllPlayers().stream().map(p -> {
```

Comment:
1) This probably should respect the vanilla behavior of now sending the entire list of players, as well as actually randomising it
2) This needs to respect the name anonymisation setting added in recent versions

Reply:
1. Where can I see the vanilla behavior for that?
2. is that [this](https://github.com/PaperMC/Velocity/blob/86c65f3910a0c1433a1741d5b6408a1508738047/proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/ClientSettings.java#L38)?

Reply:
1) in the vanilla server code, inside of MinecraftServer#tickServer; but basically they just sublist the online player list from (accounting for sample size and online count, etc), doesn't need to follow this exact behaviour, just shouldn't really be sending the full list and should be shuffled for some level of randomness
2) yes
3) In general, this should probably have a config option, not sure what others think on that

Reply:
All three of these are fixed btw